rooms-quickstart/rooms.py

```diff
@@ -53,58 +54,54 @@ def update_room(self, room_id:str):
         except HttpResponseError as ex:
             print(ex)
 
-    def update_participants(self, room_id:str, participants_list:list):
+    def get_room(self, room_id:str):
         try:
-            participants = []
-            for p in participants_list:
-                participants.append(RoomParticipant(communication_identifier=CommunicationUserIdentifier(p), role=RoleType.ATTENDEE))
-            self.rooms_client.update_participants(room_id=room_id, participants=participants)
-            print('\nRoom participants updated.')
+            get_room = self.rooms_client.get_room(room_id=room_id)
+            self.print_room(get_room)
         except HttpResponseError as ex:
             print(ex)
 
-    def get_room_after_participants_update(self, room_id:str):
+    def add_or_update_participants(self, room_id:str, participants_list:list):
         try:
-            get_room = self.rooms_client.get_room(room_id=room_id)
-            self.print_room(get_room)
+            participants = []
+            for p in participants_list:
+                participants.append(RoomParticipant(communication_identifier=CommunicationUserIdentifier(p), role=ParticipantRole.ATTENDEE))
+            self.rooms_client.add_or_update_participants(room_id=room_id, participants=participants)
+            print('\nRoom participants added or updated.')
         except HttpResponseError as ex:
             print(ex)
 
+    def list_all_rooms(self):
+        rooms = self.rooms_client.list_rooms()
+        print('\nList all valid rooms')
```

Comment:
nit: shall we call valid or active? I am using active in js and CLI but ensure we are aligned.

Reply:
Sure, I will switch to active :)